src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -41,9 +41,9 @@ internal partial class BoundCall : IInvocationExpression
     {
         IMethodSymbol IInvocationExpression.TargetMethod => this.Method;
 
-        IOperation IInvocationExpression.Instance => this.Method.IsStatic ? null : this.ReceiverOpt;
+        IOperation IInvocationExpression.Instance => (this.Method == null || this.Method.IsStatic) ? null : this.ReceiverOpt;
```

Comment:
In C#, when symbols are compared with null, they should be converted to object. The motivation is to bypass a call for an overloaded equality operator.